CMakeLists.txt

```
@@ -18,6 +18,8 @@ cmake_minimum_required(VERSION 3.3.2)
 
 project(falco)
 
+option(FALCO_Coverage "Build test suite with coverage information" OFF)
```

```suggestion
option(FALCO_COVERAGE "Build test suite with coverage information" OFF)
```

Just for conformance with other option I would personally prefer having the option variable all caps. Just nitpicking you know me 🕺 😄 but I think it can come handy not having to remember the particular case :)